src/utils/url.ts

```diff
@@ -56,11 +56,9 @@ export class Url extends URL {
   }
 
   public hasContentApiParameters() {
-    return this.search
-      .slice(1)
-      .split('&')
-      .map((parameterWithValue) => parameterWithValue.split('=')[0])
-      .some((queryParameter) => contentApiParameters.includes(queryParameter))
+    return Array.from(this.searchParams.keys()).some((key) =>
```

Comment:
Checking all params is probably not needed but this seems saver and easier to read overall.